cli_tools/google-osconfig-agent/packaging/build_packages_deb.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+trap "{ echo 'build failed' && exit 1 }" EXIT
```

Comment:
needs a semicolon after `exit 1` or you'll get a syntax error
trap on exit will always be executed -- this makes the script a permanent failing script

Reply:
ack, this was 'working' but with the syntax error, I'm working on a better version that doesn't wok just because of ordering